charts/clearblade-monitoring/charts/cb-haproxy/templates/haproxy-deployment.yaml

```diff
@@ -36,23 +36,20 @@ spec:
           emptyDir: {}
       serviceAccountName: {{ .Values.global.gcpGSMServiceAccount }}
       initContainers:
-        - name: init
-          image: google/cloud-sdk:slim
-          command: ["/bin/bash"]
-          args:
-            - -c
-            - |
-              certificates="$(gcloud secrets versions access "latest" --secret={{ default "clearblade" .Values.global.namespace }}_tls-certificates)"
```

Comment:
why is this removed? We don't support user provided certificates in gcp secrets anymore?

Reply:
I replaced this with the `pull-certificates` container defined below. This container uses the cb_controller to do the pulling/decoding of the certs. The behavior should be the same, but it pushes all of the logic into the controller plus allows this container to support both the new and old secret format.